### PR TITLE
[DEV APPROVED] - 7930 - Global Nav Visual updates

### PR DESCRIPTION
**Pull request by johnplayer1982**
## Global nav visual updates

- Slide in navigation should be full height
- Slide in navigation should only be 80 to 90% the width of the screen
- User should be able to click/press on the remaining area to close the navigation

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1664)

### CODE REVIEW

File: app/assets/stylesheets/components/common/_mobile_nav.scss [hunk 1]
```diff
@@ -96,3 +96,19 @@
...
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: none;
+  background-color: rgba(0,0,0,.5);
+
+  &.is-active {
+    display: block;
+  }
+}
```
Comment (davidtrussler) [inline app/assets/stylesheets/components/common/_mobile_nav.scss]: This needs to be only displayed at the smaller viewport.

File: app/assets/stylesheets/components/common/_global_nav.scss [hunk 2]
```diff
@@ -177,7 +193,7 @@
   @include respond-to($mq-m) {
     display: block;
     padding: $baseline-unit*2 $baseline-unit;
-    font-weight: 700;
+
```
Comment (davidtrussler) [inline app/assets/stylesheets/components/common/_global_nav.scss]: Can we keep this font weight on desktop? (was part of the last visual improvement, but I think the lighter font is better on mobile).

**Comment by davidtrussler** [comment]
👍 All looks good!

**Comment by tomas-stefano** [comment]
👍

File: app/assets/javascripts/components/GlobalNav.js [hunk 3]
```diff
@@ -64,6 +66,14 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities'], function($
     this.$globalSubNavHeading.click(function() {
       self._toggleMobileSubNav(this);
     });
+
+    this.$mobileNavClose.click(function(){
+      self._closeMobileNav();
+    });
+
+    $('[data-dough-mobile-nav-overlay]').click(function(){
```
Comment (jongilbraith) [inline app/assets/javascripts/components/GlobalNav.js]: How come this doesn't use `this.$mobileNavOverlay`?

Comment (johnplayer1982) [inline app/assets/javascripts/components/GlobalNav.js]: Good spot ill update this

**Comment by jongilbraith** [comment]
👍

**Pull request merged**